test/browser/playwright.config.ts

```diff
@@ -21,6 +21,7 @@ const config: PlaywrightTestConfig = {
   workers: process.env.CI ? 2 : undefined,
   forbidOnly: !!process.env.CI,
   reporter: 'list',
+  fullyParallel: true,
```

Comment:
@kettanaito  it's here :D

Reply:
Btw bare in mind that changing this way of running tests may unveil some setup-teardown issues that weren't caught before, but in long run I think it's crucial to have tests independent and it's better to find out about such problems/blockers sooner

Reply:
I absolutely agree. Looks like a great addition to me.